### ISSUE TITLE
Use CURL_TIMEOUT instead of CURL_CONNECTTIMEOUT

### ISSUE THREAD

**Issue by shtse8** [ptcong/easyrequest#6]
We will wait indefinitely if we can connect to the proxy but the proxy get no response from the destination.
So I prefer to use CURL_TIMEOUT instead of CURL_CONNECTTIMEOUT to limit the curl max execution time and make sure this class can get return and will not wait endlessly by changing the following code:

```
 CURLOPT_CONNECTTIMEOUT => $options['timeout'],
```

to 

```
 CURLOPT_TIMEOUT        => $options['timeout'],
```

**Comment by ptcong** [comment]
What happened if user want to upload or download a large file ? 
It may caused problem if we can't guess how many seconds needed in this case.
I think we should make new option for execution timeout (disabled by default) and keep connect time out option as current.

**Comment by shtse8** [comment]
Yes, I think we could make one more option. But the name of the options should match the curl's one for better understanding.
